src/scales/scale.category.js

```diff
@@ -119,7 +121,7 @@ module.exports = function(Chart) {
 				value = Math.round(pixel / valueDimension);
 			}
 
-			return value;
+			return value + me.minIndex;
```

Comment:
I don't think there is a ticket for it. `getValueForPixel()` is only called from test code.